ext_localconf.php

```diff
@@ -32,8 +32,17 @@
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tcemain.php']['clearCachePostProc'][] = 'FluidTYPO3\Flux\Backend\TceMain->clearCacheCommand';
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tstemplate.php']['includeStaticTypoScriptSources']['flux'] = 'FluidTYPO3\Flux\Backend\TypoScriptTemplate->preprocessIncludeStaticTypoScriptSources';
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['extTablesInclusion-PostProcessing']['flux'] = 'FluidTYPO3\Flux\Backend\TableConfigurationPostProcessor';
-$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['cms/layout/class.tx_cms_layout.php']['tt_content_drawItem']['flux'] = 'FluidTYPO3\Flux\Backend\Preview';
 
+$previewClassPostfix = '';
+$version = ((float) substr(TYPO3_version, 0, 3));
+switch (TRUE) {
```

Comment:
Depending on the above comments this may be a non-issue, but I should mention it anyway:

The naming of this class is invalid and likely to cause problems with code detecting wether or not a class name uses namespaces. Usually, we replace the implementations of classes by using the "class name registry" which allows overruling the implementation that ObjectManager will load for each class that can be loaded.

On a side note, in this case I think it would be preferable to have a "Legacy" class or sub-scope and fall back to it if the version is LTS, otherwise use the default class. Since from now on we'll only ever be requiring support for most recent LTS and most recently released iteration. But avoiding splitting the classes would be preferred ;)